js/formValidation.js

```diff
@@ -0,0 +1,43 @@
+Fliplet.Env.set('appId', 1);
+
+var android;
```

Comment:
android? give more context to this variables please :) I don't know what this means and what value could have.